packages/orbit-components/src/Pagination/Pagination.stories.tsx

```diff
@@ -99,3 +100,58 @@ export const Playground = () => {
     />
   );
 };
+
+export const Rtl = () => {
+  const pageCount = number("pageCount", 6);
+  const selectedPage = number("selectedPage", 2);
+  const hideLabels = boolean("hideLabels", false);
+  const size = select("size", Object.values(SIZES), SIZES.NORMAL);
+  const labelProgress = text("labelProgress", "2 מתוך 6");
+
+  return (
+    <RenderInRtl>
+      <Pagination
+        labelPrev="Prev"
+        labelNext="Next"
+        labelProgress={labelProgress}
```

Comment:
Would it make sense to make it 
```suggestion
        labelProgress={<div dangerouslySetInnerHTML={{ __html: labelProgress}} />}
```
as it's supposed to accept a React node?

I guess it's not great from security point of view (but since Storybook is hosted on external domains, it should be okay). Otherwise, we could just make a note to run this locally to try with additional markup elements to specify the direction of the text. 🤔

Reply:
I think this is also an issue across multiple components that accept `ReactNode` so maybe it would make sense to think about it in general. If we go the `dangerouslySetInnerHTML` way we’ll also need to disable html escaping for knobs I guess. Not sure if it’s possible to disable it per knob.

Another option could be to provide several predefined examples.

Reply:
I think it would be simpler either to leave it as it is (the story represents RTL and the default value proves it) or provide a list of 2 or 3 predefined examples as @vbulant mentioned. I wouldn't complicate things by allowing you to type raw HTML in a Storybook.

WDYT?